### ISSUE TITLE
Move plotted images to storage location and optionally remove the working directory

### ISSUE THREAD

**Issue by EdwardSafford-NOAA** [NOAA-EMC/obs-monitor#43]
Move the plotted images to an `$OM_OUTPUT` location defined in `Obs-Monitor/parm/OM_config`.  For the moment the existing directory structure (i.e. `line_plots/$model/[ozn|rad|con|min]/[summary|time|etc]`) will be preserved.  We will undoubtedly change this in future releases but the purpose here is just to move the plots to a designated storage location outside of the working directory.

That sets up part 2 -- optionally remove the working directory.  The switch is already defined by `$KEEPDATA` in the J-job script.  This setting should be added to `OM_config`. Removal of the working directory must be done by the plotting job (which is submitted by `scripts/exobsmon_plot.sh`).

**Comment by EdwardSafford-NOAA** [comment]
I've added a cleanup file to ush which is submitted as a separate job with a dependency to run after the plot job successfully completes.  That's working on `wcoss2 `and `hera`.  

It was working on `hercules` prior to yesterday's maintenance; now I'm hitting a problem with the python environment which generates this error:

` TypeError: Cannot convert numpy.ndarray to numpy.ndarray`

So something changed and it looks like a numpy version conflict.  Groovy.  I've tried rebuilding the environment but that has not altered the situation.  The struggle continues.

Update:  Looks like it's an issue with the datetime encoding.  The error occurs when cycle time is added to the dataset and I'm able to reproduce the error running an eva test file.  Digging into that now.
